### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lukasschor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
-# Gnosis Safe apps SDK
+# Safe Apps SDK
 
-[![Logo](https://raw.githubusercontent.com/gnosis/safe-apps-sdk/master/assets/logo.png)](https://gnosis.pm/)
 
 [![Build Status](https://travis-ci.org/gnosis/safe-apps-sdk.svg?branch=master)](https://travis-ci.org/gnosis/pm-contracts)
 
-Software development kit to integrate a third-party app with Safe Multisig (https://gnosis-safe.io/app/).
+Software development kit to integrate third-party appas (Safe Apps) with Safe Multisig (https://gnosis-safe.io/app/).
```
Comment (rmeissner) [inline README.md:6]: appas?

File: README.md [hunk 2]
```diff
@@ -23,21 +22,21 @@ yarn build
 
 ## Documentation
 
-This library expose a single method called initSdk that receive a single parameter, the safe multisig app URL. 
+This library exposes a single method called initSdk that receives a single parameter, the Gnosis Safe Multisig app URL.
```
Comment (rmeissner) [inline README.md:25]: \`initSdk\`

File: README.md [hunk 3]
```diff
@@ -23,21 +22,21 @@ yarn build
...
+This library exposes a single method called initSdk that receives a single parameter, the Gnosis Safe Multisig app URL. 
 
 * rinkeby: https://rinkeby.gnosis-safe.io/
 * mainnet: https://gnosis-safe.io/
 
 ```js
 const [appsSdk] = useState(initSdk('https://rinkeby.gnosis-safe.io/'));
 ```
-It returns a SKD instance that allows you to interact with the safe-multisig app.
+It returns a SDK instance that allows you to interact with the Safe Multisig application.
```
Comment (rmeissner) [inline README.md:33]: When is it Safe and when Gnosis Safe? (above it is Gnosis Safe)

File: README.md [hunk 4]
```diff
@@ -77,30 +76,30 @@ txs = [
...
-When your app is live, you can import it to Safe-multisig app. To do so, you should select the Tabs app:
+When your app is live, you can import it to the Safe Multisig application. To do so, you should select the "Apps" tab:
 
 ![alt text][safeAppsTab]
 
-[safeAppsTab]: https://raw.githubusercontent.com/gnosis/safe-apps-sdk/master/assets/safe-tab-apps.png "Safe apps tab"
+[safeAppsTab]: https://raw.githubusercontent.com/gnosis/safe-apps-sdk/master/assets/safe-tab-apps.png "Safe Multisig: Apps tab"
 
-And then from the `Manage Apps` button link add your app:
+And use the `Manage Apps` button and add add your app using a link:
```
Comment (rmeissner) [inline README.md:98]: I would remove the `And` at the beginning-

File: README.md [hunk 5]
```diff
@@ -22,13 +22,13 @@ yarn build
...
 
-This library exposes a single method called initSdk that receives a single parameter, the Gnosis Safe Multisig app URL. 
+This library exposes a single method called `initSdk` that receives a single parameter, the Safe Multisig app URL. 
 
 * rinkeby: https://rinkeby.gnosis-safe.io/
 * mainnet: https://gnosis-safe.io/
 
 ```js
-const [appsSdk] = useState(initSdk('https://rinkeby.gnosis-safe.io/'));
+const [appsSdk] = useState(('https://rinkeby.gnosis-safe.io/'));
```
Comment (rmeissner) [inline README.md:31]: what happened here? :D

Comment (lukasschor) [reply README.md:31]: 🤦

**Review by rmeissner — APPROVED** [review]

**Pull request merged**